lib/android-tools-info.ts

```diff
@@ -306,56 +304,35 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return selectedAppCompatVersion;
 	}
 
-	private async getLatestValidAndroidTarget(): Promise<string> {
-		let installedTargets = await this.getInstalledTargets();
+	private getLatestValidAndroidTarget(): string {
+		let installedTargets = this.getInstalledTargets();
 		return _.findLast(AndroidToolsInfo.SUPPORTED_TARGETS.sort(), supportedTarget => _.includes(installedTargets, supportedTarget));
 	}
 
 	private parseAndroidSdkString(androidSdkString: string): number {
 		return parseInt(androidSdkString.replace(`${AndroidToolsInfo.ANDROID_TARGET_PREFIX}-`, ""));
 	}
 
-	private async getInstalledTargets(): Promise<string[]> {
+	private getInstalledTargets(): string[] {
 		if (!this.installedTargetsCache) {
 			try {
```

Comment:
I think we don't need this `try/catch`. You can move the logger.trace in the `if` instead of `throw new Error`and do the rest in the `else`.